webroot/css/layouts/default.css

```diff
@@ -795,4 +795,11 @@ html[dir="rtl"] .md-button-right input {
 html[dir="rtl"] .md-button-right .md-button {
     left: -8px;
     right: initial;
+}
+
+.empty-info-text {
+    text-align: center;
+    font-size: 2em;
+    color: #666666;
+    margin: 50px 0 50px 0;
```

Comment:
`default.css` contains rules that are potentially applied on any page, such as style of re-usable blocks. As such, it is included on *every page* and we want to keep it free of unnecessary rules. Your rule `.empty-info-text` however is only used by the page `vocabulary/add_sentences`. For such rules, we use per-page CSS file named after the page name: `webroot/css/vocabulary/add_sentences.css`. You can move it there. :slightly_smiling_face: